ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -685,6 +714,9 @@ wiredtiger_extension_init(WT_CONNECTION *connection, WT_CONFIG_ARG *config)
         return ret != 0 ? ret : EINVAL;
     }
 
+    // Set up statistics.
```

Comment:
This comment is also redundant (same for GCP).

Reply:
removed [fcb039f](https://github.com/wiredtiger/wiredtiger/pull/8932/commits/fcb039f1e2c4c0dbf1e68bb8dad0aff3bebc9f24)